building/build-debs/homeworld-admin-tools/build-package.sh

```diff
@@ -2,23 +2,13 @@
 set -e -u
 rm -f resources/GIT_VERSION
 
-git update-index --refresh -q
 set +e
-git diff-index --quiet HEAD --
-GIT_DIFF_RETVAL=$?
-set -e
-
-GIT_ROOT="$(git rev-parse --show-toplevel)"
-set +e
-u="$(git ls-files --exclude-standard --others "${GIT_ROOT}")" && test -z "$u"
-GIT_LS_RETVAL=$?
+u="$(git status --porcelain)" && test -z "$u"
+GIT_DIRTY=$?
```

Comment:
You realize you can just set `u` here and then use `test -z "$u"` down below (or its negation, perhaps) instead of comparing the retval, right?